tagbot/action/repo.py

```diff
@@ -46,14 +56,17 @@ def __init__(
         ssh: bool,
         gpg: bool,
         lookback: int,
+        github_kwargs: Optional[Dict[str, Any]] = None,
     ) -> None:
+        if github_kwargs is None:
+            github_kwargs = dict()
```

Comment:
```suggestion
            github_kwargs = {}
```

If you want, you could also remove this and just make `{}` the default value of `github_kwargs`.

Reply:
Having `github_kwargs: Dict[str, object] = {}` leads to warning about having mutable default value, so I'm not sure if I want to do this or not.

Reply:
Ah, well I don't like warnings either. Updating the value here works for me 🙂